index.bs

```diff
@@ -138,7 +138,16 @@ Inherited: no
     :: No effect. This provides no extra information or hints to the user-agent.
 
     : <dfn export>auto</dfn>
-    :: The element [=gains containment=]. If the element is [=off-screen=], it is [=skipped=].
+    :: The element [=gains containment=]. 
+         If the following conditions hold then the element is also [=skipped=]:
+           * The element is [=off-screen=].
+           * Neither the element nor any of its [=subtree=] elements are
```

Comment:
Can you link to the HTML spec concept for focus?

Reply:
Done.